superset/viz.py

```diff
@@ -1737,7 +1737,14 @@ def query_obj(self):
         return qry
 
     def get_data(self, df: pd.DataFrame) -> VizData:
-        df.columns = ["source", "target", "value"]
+        df.rename(
+            columns={
+                self.form_data["groupby"][0]: "source",
+                self.form_data["groupby"][1]: "target",
+                df.columns[-1]: "value",
+            },
+            inplace=True,
+        )
```

Comment:
How about

```python
        source, target = self.form_data["groupby"]
        metric = self.form_data["metric"]
        df.rename(
            columns={
                source: "source",
                target: "target",
                metric: "value",
            },
            inplace=True,
        )
```

Might want to do this for other network viz as well, such as `DirectedForceViz` and `ChordViz` below.

Reply:
`self.form_data["metric"]` does not seem to always be a string (e.g. if the metric is custom defined like `COUNT(col_1)`).

Reply:
It seems the processed fields `self.groupby` and `self.metric_labels` might be more appropriate. Just wanted to be consistent how you get the original column names---either both from meta data, or both from the final df.

Reply:
@ktmud I altered it to use `self.groupby` and `self.metric_labels` (thanks for tip!)

I think I'll keep this PR specific to `SankeyViz`. I looked briefly into `ChordViz` and it looks like ChordViz uses a different metadata format, so this change might not be necessary.